apps/labs/posts/free-threaded-python-rollout.md

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Free-threaded CPython is ready to experiment with!'
+authors: [ralf-gommers]
+published: July 11, 2024
+description: 'An overview of the ongoing efforts to improve and roll out support for free-threaded CPython throughout the Python open source ecosystem'
+category: [Community, PyData ecosystem]
+featuredImage:
+  src: /posts/packaging_painpoints_in_2021/blog_feature_org.png
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/packaging_painpoints_in_2021/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone'
+---
+
+First, a few announcements:
+
+Yesterday, [py-free-threading.github.io](https://py-free-threading.github.io/) launched!
+It's both a resource with documentation around adding support for free-threaded
+Python, and a status tracker for the rollout across open source projects in the
+Python ecosystem. We hope and expect both of these to be very useful, with the
+status tracker providing a one-stop-shop to check the support status of the
+dependencies of your project (e.g., "what was the first release of a package on
+PyPI to support free-threaded Python?" or "are there nightly wheels and where
+can I find them?") and get an overview of ecosystem-wide progress.
+
+![Tracker for package compatibility with free-threaded CPython](/posts/py_free_threading_tracker.png)
```

Comment:
Not sure if it matters for the real site, but the image doesn't render in the preview.

Reply:
Yeah I'll need to fix that - the rendering does work, so I did something wrong here - I'll figure it out